### PR TITLE
Update AWS authentication.md

### PR DESCRIPTION
**Pull request by tr0njavolta**
https://github.com/upbound/docs/issues/271

### CODE REVIEW

File: content/providers/provider-aws/authentication.md [hunk 1]
```diff
@@ -429,15 +429,15 @@ The following is a full example trust policy.
 }
 ```
 
-### Create a ControllerConfig
+### Create a RuntConfig
```
Comment (tr0njavolta) [inline content/providers/provider-aws/authentication.md:432]:
```suggestion
### Create a RuntimeConfig
```

**Comment by tr0njavolta** [comment]
Thanks for reporting @haarchri - take a look and let me know if I can add anything else.

**Comment by haarchri** [comment]
runtimeConfigs little bit different - one example - we need to add the Annotation for serviceAccountTemplate - rest can be ignored - i can provide a full working example if it helps https://github.com/crossplane/crossplane/blob/master/test/e2e/manifests/pkg/deployment-runtime-config/setup/deployment-runtime-config.yaml#L6

**Comment by tr0njavolta** [comment]
> runtimeConfigs little bit different - one example - we need to add the Annotation for serviceAccountTemplate - rest can be ignored - i can provide a full working example if it helps https://github.com/crossplane/crossplane/blob/master/test/e2e/manifests/pkg/deployment-runtime-config/setup/deployment-runtime-config.yaml#L6

@haarchri please provide a full working example. I'm unclear what needs to be changed here.

**Comment by haarchri** [comment]
```
apiVersion: pkg.crossplane.io/v1beta1
kind: DeploymentRuntimeConfig
metadata:
  name: irsa-controllerconfig
spec:
  serviceAccountTemplate:
    metadata:
      annotations:
        eks.amazonaws.com/role-arn: arn:aws:iam::622346257358:role/my-custom-role
```

**Review by tr0njavolta — COMMENTED** [review]
Add deploymentruntimeconfig example update.

File: content/providers/provider-aws/authentication.md [hunk 2]
```diff
@@ -344,35 +344,35 @@ spec:
...
-with the desired permissions. 
+with the desired permissions.
 
 Configuring IRSA with the AWS Provider requires:
-* an AWS 
+* an AWS
 [IAM OIDC Provider](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html)
 * an AWS IAM Role with an editable [trust policy](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_terms-and-concepts.html#term_trust-policy)
-* a ControllerConfig to add an annotation on the AWS Provider service account
+* a RuntimeConfig to add an annotation on the AWS Provider service account
```
Comment (haarchri) [inline content/providers/provider-aws/authentication.md:359]: DeploymentRuntimeConfig

File: content/providers/provider-aws/authentication.md [hunk 3]
```diff
@@ -429,40 +429,42 @@ The following is a full example trust policy.
...
 
-### Create a ControllerConfig
+### Create a RuntimeConfig
 
 IRSA relies on an annotation on the service account attached to a pod to
-identify the IAM role to use. 
+identify the IAM role to use.
 
-Crossplane uses a ControllerConfig to apply settings to the provider, including 
+Crossplane uses a RuntimeConfig to apply settings to the provider, including
```
Comment (haarchri) [inline content/providers/provider-aws/authentication.md:437]: DeploymentRuntimeConfig

File: content/providers/provider-aws/authentication.md [hunk 4]
```diff
@@ -429,40 +429,42 @@ The following is a full example trust policy.
...
 the provider service account.
 
-Create a {{<hover label="cc" line="2">}}ControllerConfig{{</hover>}} object to
-apply a custom annotation to the provider service account. 
+Create a {{<hover label="cc" line="2">}}RuntimeConfig{{</hover>}} object to
```
Comment (haarchri) [inline content/providers/provider-aws/authentication.md:440]: DeploymentRuntimeConfig

File: content/providers/provider-aws/authentication.md [hunk 5]
```diff
@@ -429,40 +429,42 @@ The following is a full example trust policy.
...
-spec: {}    
+spec:
+  serviceAccountTemplate:
+    metadata:
+      annotations:
+        eks.amazonaws.com/role-arn: arn:aws:iam::622346257358:role/my-custom-role
 ```
 
-### Apply the ControllerConfig
+### Apply the RuntimeConfig
```
Comment (haarchri) [inline content/providers/provider-aws/authentication.md:462]: DeploymentRuntimeConfig

File: content/providers/provider-aws/authentication.md [hunk 6]
```diff
@@ -429,40 +429,42 @@ The following is a full example trust policy.
...
 
-Install or update the provider with a 
-{{<hover label="provider" line="7">}}controllerConfigRef{{</hover>}} with the
-{{<hover label="provider" line="8">}}name{{</hover>}} of the 
-{{<hover label="cc" line="4">}}ControllerConfig{{</hover>}}.
+Install or update the provider with a
+{{<hover label="provider" line="7">}}runtimeConfigRef{{</hover>}} with the
+{{<hover label="provider" line="8">}}name{{</hover>}} of the
+{{<hover label="cc" line="4">}}RuntimeConfig{{</hover>}}.
```
Comment (haarchri) [inline content/providers/provider-aws/authentication.md:467]: DeploymentRuntimeConfig

File: content/providers/provider-aws/authentication.md [hunk 7]
```diff
@@ -471,19 +473,19 @@ metadata:
...
-    name: irsa-controllerconfig
+  runtimeConfigRef:
+    name: irsa-runtimeconfig
 ```
 
-After the provider finishes installing verify Crossplane applied the 
+After the provider finishes installing verify Crossplane applied the
 {{<hover label="sa" line="5">}}annotation{{</hover>}}
-on the service account from the ControllerConfig. 
+on the service account from the RuntimeConfig.
```
Comment (haarchri) [inline content/providers/provider-aws/authentication.md:482]: DeploymentRuntimeConfig

File: utils/vale/styles/Google/Headings.yml [hunk 8]
```diff
@@ -77,6 +77,8 @@ exceptions:
   - ProviderConfigs
   - Providers
   - REPL
+  - RuntimeConfig
```
Comment (haarchri) [inline utils/vale/styles/Google/Headings.yml:80]: DeploymentRuntimeConfig

File: utils/vale/styles/Google/Headings.yml [hunk 9]
```diff
@@ -77,6 +77,8 @@ exceptions:
...
+  - RuntimeConfigs
```
Comment (haarchri) [inline utils/vale/styles/Google/Headings.yml:81]: DeploymentRuntimeConfigs

**Review by haarchri — APPROVED** [review]
Please Check my latest comments

**Pull request merged**